cluster/kubernetes/resourcekinds.go

```diff
@@ -86,13 +86,13 @@ func (dk *deploymentKind) getPodControllers(c *Cluster, namespace string) ([]pod
 
 	var podControllers []podController
 	for _, deployment := range deployments.Items {
-		podControllers = append(podControllers, makeDeploymentPodController(&deployment))
+		podControllers = append(podControllers, makeDeploymentPodController(deployment))
```

Comment:
Or, ftr, take the address of the slice item rather than the variable.

Reply:
That's a nicer fix, I'll do that.